server/src/main/java/org/opensearch/identity/authz/PERMISSIONS_MODEL.md

```diff
@@ -0,0 +1,52 @@
+# Permissions Model - Native AuthZ
+
+Addresses the following [Identity Roadmap](../../../../../../../../IDENTITY_ROADMAP.md) Authorization items:
+
+> - [Complexity 3] Design the permissions model and any potential changes. Notably existing permissions based on action names will be on deprecation path
+> - [Complexity 8] For the new permissions model come up with structured naming convention building blocks, such as {SourceName}{ResourceName}{ActionVerb} -> Ad.Detector.Create.  Additionally come up with a 'blessed' list of action verbs to prevent http methods from being used as action names.
+
+### Permissions Model
+
+In order to align with the Identity-focused changes being made to OpenSearch, the way that Permissions are handled requires modification.
+Prior to the Identity changes, Permissions were conceptualized as floodgates which controlled access to a given action that an OpenSearch cluster could perform.
+The pre-Identity permissions model made common use of a self-describing nomenclature which saw many permissions correlate to REST API operations.
+Despite this correlation, many REST API operations make use of numerous permissions at a time making the control of API access insufficient.
+
+With the transition to an Identity-based OpenSearch, it is productive to re-conceptualize Permissions as tokens which grant subjects increased privileges to certain data and operations.
+Keeping with the general practice of granting "least privileges" Permissions can be renovated to associate with a given subject Identity instead of being based on a cluster operation.
+Instead of Permissions primarily serving to restrict the certain operations on a cluster, the new Permission model seeks to grant individual Subject the privileges to execute operations.
```

Comment:
Nit: `... on a cluster, this new Permission model seeks to individually grant operation-execution privileges to each Subject.`

Reply:
This is more clear--good suggestion!